File: app/src/main/java/com/lifedawn/bestweather/ui/weathers/detailfragment/dto/GridItemDto.java
package com.lifedawn.bestweather.ui.weathers.detailfragment.dto;

import android.graphics.drawable.Drawable;

import androidx.annotation.Nullable;

public class GridItemDto {
	public final String label;
	public final String value;
	public final Drawable img;
	public Integer imgRotate;
	
	public GridItemDto(String label, String value, @Nullable Drawable img) {
		this.label = label;
		this.value = value;
		this.img = img;
	}

	public GridItemDto(String label, String value, Drawable img, Integer imgRotate) {
		this.label = label;
		this.value = value;
		this.img = img;
		this.imgRotate = imgRotate;
	}

	public GridItemDto setImgRotate(int imgRotate) {
		this.imgRotate = imgRotate;
		return this;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/ui/weathers/detailfragment/dto/GridItemDto.kt
package com.lifedawn.bestweather.ui.weathers.detailfragment.dto

import android.graphics.drawable.Drawable

class GridItemDto {
    val label: String
    val value: String
    val img: Drawable?
    var imgRotate: Int? = null

    constructor(label: String, value: String, img: Drawable?) {
        this.label = label
        this.value = value
        this.img = img
    }

    constructor(label: String, value: String, img: Drawable?, imgRotate: Int?) {
        this.label = label
        this.value = value
        this.img = img
        this.imgRotate = imgRotate
    }

    fun setImgRotate(imgRotate: Int): GridItemDto {
        this.imgRotate = imgRotate
        return this
    }
}